.mergify.yml

```
@@ -16,9 +16,4 @@ pull_request_rules:
       - author=imgbot[bot]
     actions:
       merge:
-        method: merge
-  - name: delete head branch on merged pull requests
-    conditions:
-      - merged
-    actions:
-      delete_head_branch:
\ No newline at end of file
+        method: merge
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)